lib/jekyll/commands/new_theme.rb

```diff
@@ -1,33 +1,36 @@
 require "erb"
+module Jekyll
+  module Commands
+    class NewTheme < Command
```

Comment:
I was trying to avoid all the indentation here. We're going to move to using autoload, which should take care of not having Command.